### DISCUSSION TITLE
How to clear the chat context?

### DISCUSSION THREAD

**Discussion by N8python** [withcatai/node-llama-cpp#98] [Q&A] [answered]
I want to load the context of a model once, query it with a prompt, and then be able to query it again without the model remembering the first prompt. Is this doable with the current library API?

**Comment by giladgd** [comment, marked as answer]
@N8python This is not possible ATM, but it'll be possible as part of #85 that I'm working on right now

**Reply by N8python** [reply]
Awesome! Is there a workaround I could use in the meantime, bar loading the entire Llama context again?

**Reply by giladgd** [reply]
@N8python If this is still relevant, see #125 for a workaround

**Reply by N8python** [reply]
Thank you!
